src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1008,8 +1006,13 @@ end
     filter(fun, df::AbstractDataFrame; view::Bool=false)
     filter(cols => fun, df::AbstractDataFrame; view::Bool=false)
 
-Return a data frame containing only rows from `df` for which `fun`
-returns `true`.
+Return a data frame containing only rows from `df` for which `fun` returns
+`true`.
+
+The use of this function is discouraged as it has API that is inconsistent with
+other transformation functions like [`select`](@ref). Instead it is recommended
+to use the [`subset`](@ref) function. The [`filter`](@ref) function is supported
+to ensure backward compatibility.
```

Comment:
Same here (and below). Mentioning that `subset` is faster should be enough IMO.

Reply:
done